### DISCUSSION TITLE
Missing websites

### DISCUSSION THREAD

**Discussion by majant0** [linuxserver/Heimdall#1205] [Q&A]
I just installed Heimdall and I'm brand new to it, so I'm sure the issue is somewhere on my end but I can't figure it out.

I am missing the regular websites I am setting up.

My home screen:
<img width="1213" alt="image" src="https://github.com/linuxserver/Heimdall/assets/95008455/ac60405e-d516-4a33-b58d-06605d89c2c8">

My App List:
<img width="579" alt="image" src="https://github.com/linuxserver/Heimdall/assets/95008455/0b28057a-9710-4ebc-a517-a38c35005f9e">
In this example, Dashy, Homebox and Immich are set up as websites.

Config:
<img width="996" alt="image" src="https://github.com/linuxserver/Heimdall/assets/95008455/bc43a4a3-d1df-4c06-8e0e-4a0555288d65">

It seems so simple...There's little configuration so I don't know what I'm missing. Any help would be appreciated.

**Comment by lastsamurai26** [comment]
on the rights top corner there you can Find the checkmark " Pinned" this need's to be active
